src/GrumPHP/Task/Php7cc.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Php7cc task
+ */
+class Php7cc extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'php7cc';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults(array(
+            'exclude' => array('vendor'),
```

Comment:
The vendor folder shouldn't be add as a default exclude parameter.
Since you are adding the changed files to the command, the vendor folder won't be checked.
(At least if you add the vendor folder to the .gitignore file in your project)

You can also change the default configuration in the documentation file.